src/applications/vre/28-1900/containers/App.jsx

```diff
@@ -1,99 +1,35 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import { connect } from 'react-redux';
 
-import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
-import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import FormFooter from 'platform/forms/components/FormFooter';
-import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
-
-import {
-  WIZARD_STATUS_NOT_STARTED,
-  WIZARD_STATUS_RESTARTED,
-  WIZARD_STATUS_COMPLETE,
-  restartShouldRedirect,
-} from 'platform/site-wide/wizard';
 
 import formConfig from '../config/form';
 import OrientationWizardContainer from './OrientationWizardContainer';
-import { WIZARD_STATUS, VRE_ROOT_URL } from '../constants';
+import { WIZARD_STATUS } from '../constants';
+
+import FormInProgressNotification from '../components/FormInProgressNotification';
 
-function App({ location, children, router, chapter31Feature, isLoggedIn }) {
-  const [wizardState, setWizardState] = useState(WIZARD_STATUS_NOT_STARTED);
+function App({ chapter31Feature }) {
   let content;
 
   const wizardStateHandler = status => {
     sessionStorage.setItem(WIZARD_STATUS, status);
-    setWizardState(status);
   };
 
-  useEffect(() => {
```

Comment:
No longer needed here since the index route now contains the form application, which handles browser redirect.